client/containers/DashboardOverview/helpers/QuickActions.tsx

```diff
@@ -6,7 +6,7 @@ require('./quickActions.scss');
 
 export type QuickAction = {
 	icon: IconName;
-	label: React.ReactNode;
+	label: string;
```

Comment:
Super minor (feel free to ignore): at a quick glance it looks like we use string-only values for quick action labels. But I wonder if we could preserve accepting an arbitrary React.Node and instead use the `href` as the key?